ios/Pangea.xcodeproj/project.pbxproj

```diff
@@ -1041,13 +1041,13 @@
 				TargetAttributes = {
```

Comment:
Please revert all changes from this file, since it's unrelated to PR.

Reply:
I am sorry about it but I am not sure how to revert the changes in a specific file.
Can we just leave it since it does not affect too much?